pages/ja/lb4/exposing-graphql-apis.md

```diff
@@ -0,0 +1,116 @@
+---
+lang: ja
+title: 'GraphQL APIの公開'
+keywords: LoopBack 4.0, LoopBack 4, GraphQL
+sidebar: ja_lb4_sidebar
+permalink: /doc/jp/lb4/exposing-graphql-apis.html
```

Comment:
Please add `layout: translation` and update the link from `jp` to `ja`

Reply:
Added and modified `layout: translation` and `jp to ja`.

Reply:
This is still `jp` in the permalink
```suggestion
permalink: /doc/ja/lb4/exposing-graphql-apis.html
```